### PR TITLE
Fix bpd check boxes

### PR DESCRIPTION
**Pull request by Vicarious1**
Add logic to validate the following criteria before checking a BPD box:
- 350 day stake minimum
- Stake ends on or after the BPD

### CODE REVIEW

File: src/app/staking-page/staking-page.component.ts [hunk 1]
```diff
@@ -145,6 +146,11 @@ export class StakingPageComponent implements OnDestroy {
     }, this.settingsData.settings.checkerBPD);
   }
 
+  public shouldCheckBoxForBPD(index) {
+    return this.formsData.depositDays > 349 &&
```
Comment (Vicarious1) [inline src/app/staking-page/staking-page.component.ts:150]: Ideally i'd have written a unit test that validates this for each BPD, but the unit tests are broken right now. Maybe that's something I can work on at some point

**Review by AudioSin — APPROVED** [review]

**Comment by AudioSin** [comment]
@Vicarious1 Can this be merged?

**Review by Mvii7 — APPROVED** [review]
Tested, 349 days doesn't check BPD 1, dunno how it will behave after BPD 1 but is good for now

File: src/app/staking-page/staking-page.component.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ interface StakingInfoInterface {
 
 const FULL_PERIOD = 700;
 const AVAILABLE_DAYS_AFTER_END = 14;
+const SECONDS_PER_DAY = 86400000;
```
Comment (Mvii7) [inline src/app/staking-page/staking-page.component.ts:23]: Pls get this from settings

**Review by Mvii7 — CHANGES_REQUESTED** [review]

File: src/app/staking-page/staking-page.component.ts (hunk 2 above)
Comment (Mvii7) [reply src/app/staking-page/staking-page.component.ts:23]: Also its 86400 per day ¯\_(ツ)_/¯

**Review by Mvii7 — APPROVED** [review]

**Pull request merged**